receiver/otlpreceiver/otlp_test.go

```diff
@@ -495,6 +495,35 @@ func TestOTLPReceiverInvalidContentEncoding(t *testing.T) {
 	}
 }
 
+func TestOTLPReceiverNoContentType(t *testing.T) {
```

Comment:
Let me know if you'd prefer to add this to an existing test or test it differently. Bad encoding is a simple way to trigger an error, but `TestOTLPReceioverInvalidContentEncoding` tests for different things.